app/command.go

```diff
@@ -45,10 +45,11 @@ func CreateCommandPost(post *model.Post, teamId string, response *model.CommandR
 		parseSlackAttachment(post, response.Attachments)
 	}
 
-	switch response.ResponseType {
-	case model.COMMAND_RESPONSE_TYPE_IN_CHANNEL:
+	if response.ResponseType == model.COMMAND_RESPONSE_TYPE_IN_CHANNEL {
 		return CreatePost(post, teamId, true)
-	case model.COMMAND_RESPONSE_TYPE_EPHEMERAL:
+	}
+
+	if response.ResponseType == "" || response.ResponseType == model.COMMAND_RESPONSE_TYPE_EPHEMERAL {
```

Comment:
Can you make this an `else if` off the last if statement? It's a really minor thing, but I prefer to clearly show that there's no case where both branches of this are ran